scripts/update-unknown.js

```diff
@@ -8,11 +8,11 @@ async function main() {
   console.log(`${bans.length} Unknown Bans To Classify`);
   for (const ban of bans) {
     const old = ban.reason;
-    ban.reason = classifyBanReason(ban.rawReason);
     if (old !== ban.reason) {
       console.log(old, ban.reason);
       console.log(`Updating Ban: ${ban.reason}`);
       await ban.save();
+    ban.reason = classifyBanReason(ban.rawReason, ban.rawNote);
```

Comment:
Was this change intentional? It doesn't seem right, as with this change ban reasons will never be updated if they are updated on BM.

Reply:
fixed in https://github.com/CommunityBanList/Communitybanlist/pull/11/commits/239588b9eac41e7fbb045414fe98a902f321753c